src/Misc/assetsManager.ts

```diff
@@ -1013,8 +1017,8 @@ export class AssetsManager {
      * @param files defines the list of files to load (can be null)
      * @returns a new CubeTextureAssetTask object
      */
-    public addCubeTextureTask(taskName: string, url: string, extensions?: string[], noMipmap?: boolean, files?: string[]): CubeTextureAssetTask {
-        var task = new CubeTextureAssetTask(taskName, url, extensions, noMipmap, files);
+    public addCubeTextureTask(taskName: string, url: string, extensions?: string[], noMipmap?: boolean, files?: string[], prefiltered?: boolean): CubeTextureAssetTask {

```

Comment:
documentation check will fail ;)

Reply:
Added prefiltered doc item. Forgot that one before :)